### PR TITLE
Build/Test Tools: Add theme unit test data locally.

### PR DESCRIPTION
**Pull request by costdev**
The [WPTT/theme-test-data/](https://github.com/WPTT/theme-test-data/) repository is no longer available and was used by the performance tests.

This PR adds the `themeunittestdata.wordpress.xml` file directly to the `wordpress-develop` repository in `tests/performance/data/` to remove the reliance on an "external" dependency.

The contents of `themeunittestdata.wordpress.xml` were taken from [the Wayback Machine snapshot](https://web.archive.org/web/20230403052347/https://github.com/WPTT/theme-test-data/blob/master/themeunittestdata.wordpress.xml) of the file on April 3rd, 2023.

Trac ticket: https://core.trac.wordpress.org/ticket/58569

### CODE REVIEW

File: src/wp-includes/load.php [hunk 1]
```diff
@@ -213,6 +213,7 @@ function wp_check_php_mysql_versions() {
  * @return string The current environment type.
  */
 function wp_get_environment_type() {
+	sleep( 13 );
```
Comment (mukeshpanchal27) [inline src/wp-includes/load.php]: Why sleep?

Comment (peterwilsoncc) [reply src/wp-includes/load.php]: @mukeshpanchal27 That's me just testing to make sure I'm not about to commit broken tests. I'll be getting rid of all this weirdness soon. See https://github.com/WordPress/wordpress-develop/pull/4636/commits/acaabf27b64891e616601858584765281ebcc9a1

**Review by mukeshpanchal27 — COMMENTED** [review]
@costdev What happens when we receive new updates for the test data? It can potentially increase overhead, in my opinion. I suggest waiting for feedback from other team members to determine the best path forward.

**Comment by costdev** [comment]
@mukeshpanchal27 I agree that we should wait for input from other Performance and Themes team members. This was a stopgap fix to get PRs/commits working in the short term, and potentially to remove the reliance on a dependency outside the `wordpress-develop` repo in the long term if that's desired.

Regarding overhead, I guess that would partially depend on how often the test data tends to get updated. I'm not aware of the frequency of this. That said, I appreciate the benefits of having a file maintained elsewhere that can receive faster updates without PR authors needing to merge/rebase on `trunk`.

Everyone piled on to get it resolved as quickly as possible, and this PR was one of those solutions (a stopgap) in case the repo was unavailable for an extended period of time.

We don't yet know exactly what caused the repo to become unavailable, so we may consider this a blip, or we may consider whether something should change so that the test suite is more stable.

**Comment by carolinan** [comment]
The test data gets updated about once every two years. Ideally, something other than one big XML file would be used.
Like a CLI command to install different types of data, not everyone is interested in testing "classic" content, or old block markup.

**Review by joemcgill — APPROVED** [review]
The approach and file used in this PR looks good to me. Given that the repository issues have been resolved and there are active conversations going to move that repo under control of the WordPress org, I'd prefer to hold off on making this change for now to avoid having to update this file in two places in the future.

**Comment by peterwilsoncc** [comment]
Superseded by https://github.com/WordPress/wordpress-develop/pull/4668

**Pull request closed without merging**